### DISCUSSION TITLE
Unable to install

### DISCUSSION THREAD

**Discussion by ajithprasadpv** [llm-workflow-engine/llm-workflow-engine#96] [Q&A]
Getting the below error in mac terminal. Pls suggest a solution 

(base)$ pip install git+https://github.com/mmabrouk/chatgpt-wrapper
Collecting git+https://github.com/mmabrouk/chatgpt-wrapper
  Cloning https://github.com/mmabrouk/chatgpt-wrapper to /private/var/folders/xd/vbllw5n56zv6pb02ms0y63080000gp/T/pip-req-build-3k0voy7a
  Running command git clone -q https://github.com/mmabrouk/chatgpt-wrapper /private/var/folders/xd/vbllw5n56zv6pb02ms0y63080000gp/T/pip-req-build-3k0voy7a
    ERROR: Command errored out with exit status 1:
     command: /opt/anaconda3/bin/python -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/private/var/folders/xd/vbllw5n56zv6pb02ms0y63080000gp/T/pip-req-build-3k0voy7a/setup.py'"'"'; __file__='"'"'/private/var/folders/xd/vbllw5n56zv6pb02ms0y63080000gp/T/pip-req-build-3k0voy7a/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /private/var/folders/xd/vbllw5n56zv6pb02ms0y63080000gp/T/pip-pip-egg-info-im8dh49r
         cwd: /private/var/folders/xd/vbllw5n56zv6pb02ms0y63080000gp/T/pip-req-build-3k0voy7a/
    Complete output (9 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/private/var/folders/xd/vbllw5n56zv6pb02ms0y63080000gp/T/pip-req-build-3k0voy7a/setup.py", line 2, in <module>
        from chatgpt_wrapper.chatgpt import VERSION
      File "/private/var/folders/xd/vbllw5n56zv6pb02ms0y63080000gp/T/pip-req-build-3k0voy7a/chatgpt_wrapper/__init__.py", line 1, in <module>
        from .chatgpt import ChatGPT
      File "/private/var/folders/xd/vbllw5n56zv6pb02ms0y63080000gp/T/pip-req-build-3k0voy7a/chatgpt_wrapper/chatgpt.py", line 15, in <module>
        from playwright.sync_api import sync_playwright
    ModuleNotFoundError: No module named 'playwright'
    ----------------------------------------
WARNING: Discarding git+https://github.com/mmabrouk/chatgpt-wrapper. Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.

**Comment by jromal** [comment]
Just read the error:

ModuleNotFoundError: No module named 'playwright'

You need to install additioanl packages: playwright, rich, ... 
if you do not have them. 

There should be a requirements.txt list to be honest.
